components/locid/Cargo.toml

```diff
@@ -38,6 +38,7 @@ tinystr = { path = "../../utils/tinystr", version = "0.5.0", features = ["alloc"
 serde = { version = "1.0", optional = true, default-features = false, features = ["alloc"] }
 writeable = { version = "0.3", path = "../../utils/writeable" }
 displaydoc = { version = "0.2.3", default-features = false }
+zerovec = { version = "0.6", path = "../../utils/zerovec", optional = true }
```

Comment:
thought: hmm, this is the first time we're making zerovec optional. I see why — locid doesn't need it, locid's clients do. I feel like given the generally pervasive use of zerovec I'd be open to making this a mandatory dependency, but either works for me.

Reply:
Let's discuss in a separate issue: #1700